media/css/devreg/lookup-tool.styl

```diff
@@ -492,3 +492,17 @@ a:not([href]) {
         display: none;
     }
 }
+
+.group-memberships {
+    tr {
+        margin-top: 1.5em;
+        border-top: 1px dotted $border-taupe;
```

Comment:
We alphabetize CSS properties, so `border-top` should be the first line. Same issue below.